etna/collections/migrations/0006_add_record_teaser_image.py

```diff
@@ -25,7 +25,7 @@ class Migration(migrations.Migration):
         ),
         migrations.AlterField(
             model_name="resultspagerecordpage",
-            name="record_iaid",
+            name="record_metadataId",
```

Comment:
Changing existing migrations that have already run is a no-no. This change should be reverted and taken care of in a new migration